docs/utd_ganymede.md

```diff
@@ -0,0 +1,18 @@
+# nf-core/configs: UTD Ganymede Configuration
+
+All nf-core pipelines have been successfully configured for use on the Ganymede HPC cluster at the [The Univeristy of Texas at Dallas](https://www.utdallas.edu/).
+
+To use, run the pipeline with `-profile utd_ganymede`. This will download and launch the [`utd_ganymede.config`](../conf/utd_ganymede.config) which has been pre-configured with a setup suitable for the Ganymede HPC cluster. Using this profile, a docker image containing all of the required software will be downloaded, and converted to a Singularity image before execution of the pipeline.
+
+Before running the pipeline you will need to load Singularity using the environment module system on Ganymede. You can do this by issuing the commands below:
+
+```bash
+## Singularity environment modules
+module purge
+module load singularity
```

Comment:
```suggestion
module load Singularity/2.4.5
```

Reply:
For some reason our `singularity` modules are lowercase, I had to double check this myself. I'll fix this in the other place.